eng/pipelines/tools-repo-versioning.yml

```diff
@@ -16,7 +16,24 @@ jobs:
         # Remove Organization name i.e Transform Azure/azure-sdk-tools -> azure-sdk-tools before appending build Number
```

Comment:
Did you plan to split out this entire set of steps into a yml file that could be used by both azure-sdk-tools and azure-sdk-build-tools?

Reply:
Yes that was what I did initially but it required the multiple checkout.